src/external_cli/mod.rs

```diff
@@ -2,3 +2,4 @@
 
 pub mod arg_builder;
 pub(crate) mod rustup;
+pub(crate) mod wasm_bindgen;
```

Comment:
I'm not sure how accessible this is going to be, since the binary is technically a separate crate and cannot access `pub(crate)` in the library. Is there something I'm missing?

Reply:
In #24, I currently use these commands internally for the `bevy build` and `bevy run` implementations.

I'm open to changing it to `pub` though

Reply:
Nope, just making sure this wasn't a mistake! It's fine how it is :)